module/user-roles/helper.js

```diff
@@ -87,4 +87,15 @@ module.exports = class UserRolesHelper {
     });
   }
 
+  static find(query){
```

Comment:
@praveenKDass - As discussed implement spacing and indentation across all files. Also, keep naming of functions as consistent e.g. updateMany and use existing functions like roleDocuments here instead of creating a new find function.
cc - @ankitpws